README.md

```diff
@@ -146,5 +170,5 @@ and provides interactive visualization of the bounding boxes and their related l
 If you wish to disable the mini web app, start the model serving API by running:
 
 ```
-$ docker run -it -p 5000:5000 -e DISABLE_WEB_APP=true max-tf-object-detection
+$ docker run -it -p 5000:5000 -e DISABLE_WEB_APP=true max-object-detector
```

Comment:
Given this command in particular is an example I think just having one command is fine. If you think having it be `codait/max-object-detector` rather than `max-object-detector` I'm ok with that, but I think which is used is generally irrelevant in this case since the important part is the env var

Reply:
@MLnick I believe this is the only hanging review comment, if you're ok with it can this be merged?

Reply:
I decided that changing this to `codait/max-object-detector` made more sense. But I still believe it's ok with just the one example line.